GIT_VERSION

```diff
@@ -0,0 +1 @@
+1.11.0pre
```

Comment:
let's figure out how not to have the version in two places.

Reply:
This needs to be documented somewhere in our doc folder, but this is automatically injected by packit and will do the right thing, provided that release branches update the GIT_VERSION file in the branch. Given our current tagging strategy, I'm not sure there's another way. (ie: master today has no lineage with any release branch).

Reply:
Then don't use that tool.  Strong feelings here.  We can't have version numbers in a bunch of places.

Reply:
This is completely configurable in the packit config, I'll grep it out of configure.ac